recipes/signac-dashboard/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "signac-dashboard" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "bb4214e7f2be4185f7e0e1550f544b2b44f6ad04e426fedec754cf8fc9dae0a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/glotzer/signac-dashboard/get/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python >=3.4
```

Comment:
We shouldn't have `python` version dependencies like this; `conda` handles these specially, and the `skip: true  # [py<34]` bit does it. I'll remove this in my merge in a second. :)